Documentation/ceph-csi-troubleshooting.md

```diff
@@ -441,3 +441,54 @@ $ rbd ls --id=csi-rbd-node -m=10.111.136.166:6789 --key=AQDpIQhg+v83EhAAgLboWIbl
 ```
 
 Where `-m` is one of the mon endpoints and the `--key` is the key used by the CSI driver for accessing the Ceph cluster.
+
+## Node Loss
+
+When a node is lost, you will see application pods on the node stuck in the `Terminating` state while another pod is rescheduled and is in the `ContainerCreating` state.
+
+To allow the application pod to start on another node, Kubernetes requires you to force delete the pod. After a timeout (8-10 minutes), your pod will be allowed to start on another node.
+### Force delete your application pod. After some timeout (8-10 minutes), your pod will be rescheduled
+
+To force delete the pod stuck in the `Terminating` state:
+
+```console
+$ kubectl -n rook-ceph delete pod my-app-69cd495f9b-nl6hf --grace-period 0 --force
+```
+
+**And** then wait for timeout (8-10 minutes).
```

Comment:
If the watcher still not deleted the pod won't go to running state. in that case on the rbd image, we need to wait more or go and blacklist the watcher.

Reply:
added